tool/src/test/resources/org/datacommons/tool/genmcf/resolution/output/report.json

```diff
@@ -25,7 +25,8 @@
     "LEVEL_WARNING": {
       "counters": {
         "Existence_MissingReference_typeOf": "1",
-        "Existence_MissingReference_Property": "2"
+        "Existence_MissingReference_Property": "2",
+        "StatsCheck_1_Sigma": "1"
```

Comment:
Should we skip 1 Sigma? (have only 3)

Reply:
Actually, yeah, looking at prod it looks like 1 and 2 sigma get computed but the dashboard only displays 3 sigma. Updated